Taskodrome/pages/main.php

```diff
@@ -160,7 +160,25 @@ function write_bug_rows( $p_rows )
       print '<p class="version" value="'.version_get_field($ver_id, "version").'"></p>';
     }
 
+    $status_code = null;
+    $status_name = explode( ';' , $status_order );
+    $t_enum_statuses = config_get( 'status_enum_string' );
+    $t_statuses = explode( ',', $t_enum_statuses );
+    $t_status_array = array();
+    foreach( $t_statuses as $t_status  ) {
+        $t_status_array[] = explode( ':', $t_status );
+    }
+    
+    foreach( $status_name as $t_name ) {
+        for ( $i = 0 ; $i < sizeof($t_status_array) ; $i++ ){
+            if ( 0 == strcmp($t_name, $t_status_array[$i][1]) ){
```

Comment:
You're trying to compare status name from mantis ($t_status_array[$i][1], from config_get( 'status_enum_string' )), that are always english, and localized status names ($t_name, from $status_order), that may be on any other language. In such a case, this code compare will never return "true".

Reply:
@Zipher04 
Would you like to fix it?